pkg/block/s3/adapter.go

```diff
@@ -724,7 +724,7 @@ func (a *Adapter) CompleteMultiPartUpload(ctx context.Context, obj block.ObjectP
 	etag := strings.Trim(aws.ToString(resp.ETag), `"`)
 	return &block.CompleteMultiPartUploadResponse{
 		ETag:             etag,
-		ContentLength:    headResp.ContentLength,
+		ContentLength:    *headResp.ContentLength,
```

Comment:
Can you use the aws.* helper function to extract the value as it will check for nil